bedrock-entry.sh

```diff
@@ -96,17 +92,13 @@ case ${VERSION^^} in
     ;;
   *)
     # use the given version exactly
+    echo "Using given version ${VERSION}"
+    lookupVersion serverBedrockLinux "${VERSION}"
     ;;
 esac
 
 if [[ ! -f "bedrock_server-${VERSION}" ]]; then
 
-  if [[ -z "${DOWNLOAD_URL}" ]]; then
-    binPath=bin-linux
-    isTrue "${PREVIEW}" && binPath+="-preview"
```

Comment:
It looks like the handling of preview release's path will get lost with these changes.